### PR TITLE
QA tests for quoted message improvements

### PR DESCRIPTION
**Pull request by ruudniew**
1. Enable quoting messages in threads
2. Fix bug where updating a message could lead to an error when the quoted_message_id was left out
3. Make sure the quoted_message is enriched in the message.updated event

### CODE REVIEW

File: test/integration/channels.js [hunk 1]
```diff
@@ -2659,6 +2702,66 @@ describe('Quote messages', () => {
...
+			expect(updatedEvent.message.text).to.equal(msgText);
+			expect(updatedEvent.message.quoted_message).to.not.be.undefined;
+			expect(updatedEvent.message.quoted_message_id).to.equal(secondMessage.id);
+			expect(updatedEvent.message.quoted_message.id).to.equal(secondMessage.id);
+			expect(updatedEvent.message.quoted_message.text).to.equal(secondMessage.text);
+			expect(updatedEvent.message.type).to.equal('regular');
+			expect(updatedEvent.message.parent_id).to.be.undefined;
+			expect(updatedEvent.message.quoted_message.user).to.not.be.undefined;
+
+			console.log('before teardown');
```
Comment (virdesai) [inline test/integration/channels.js:2761]: is this necessary or was it just for local testing purposes?

**Review by virdesai — APPROVED** [review]
LGTM

File: test/integration/channels.js (hunk 1 above)
Comment (ruudniew) [reply test/integration/channels.js:2761]: Good catch, will remove

**Review by ferhatelmas — APPROVED** [review]
LGTM 


don't forget removing console.

Btw, we might add it to lint too @mahboubii

**Pull request merged**